src/1.2.1/develop/history.md

```diff
@@ -0,0 +1,708 @@
+# Queries and history
+
+The fundamental property of immudb is that it's an append-only database.
+This means that an update is a new insert of the same key with a new value.
+It's possible to retrieve all the values for a particular key with the history command.
+
+`History` accepts the following parameters:
+* `Key`: a key of an item
+* `Offset`: the starting index (excluded from the search). Optional
+* `Limit`: maximum returned items. Optional
+* `Desc`: items are returned in reverse order. Optional
+* `SinceTx`:
```

Comment:
* `SinceTx`: immudb will wait that the transaction specified by SinceTx is processed. Optional